target-platforms/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to target-platforms
+
+## Setting up the create-platform script
+
+This must be done once per computer. It makes the
+[create-platform](./packages/create-plaform) available as a command in your
+system.
+
+The steps are:
+
+```sh
+# Go to your local clone of liferay-frontend-projects
+$ cd ~/liferay-frontend-projects ↩
+
+# Run yarn link inside create-platform project
+$ cd target-platforms/packages/create-platform && yarn link ↩
+```
+
+## Regenerating a new platform
+
+Whenever you want to create a new platform (or regenerate an existing one) you
+must follow these steps:
+
+> 👀 Before regenerating a platform it's wise to make sure that its working copy
+> is committed and clean. This way, if any error happens, you can revert the
+> platform. Additionally, this will let you review what the `create-platform`
+> script has changed.
+
+```sh
+# Go to your local clone of liferay-portal
+$ cd ~/liferay-portal ↩
+
+# Fetch all remote tags from the `upstream` remote
+$ git fetch upstream --tags ↩
+
+# Checkout the target platform tag
+$ git checkout 7.4.0-ga1 ↩
+
+# Run yarn in modules folder to update packages at node_modules
+$ cd modules && yarn ↩
```

Comment:
We might want to make sure we run `yarn` with dxp's versions just in case there is a mismatch.

We mention something like this nom our npm-tools controbuting doc
```
env PATH=../build/node/bin:$PATH node ../build/node/lib/node_modules/yarn/yarn-1.13.0.js
```

Reply:
Don't worry too much about this: the process is so manual that it's almost impossible to do something wrong because, after running the script you can't blindly commit the resulting modifications without reviewing them in Git. In fact, you need to pass SF or lint will complain because of how JSON is formatted.

Think of the `create-platform` as a helper to do 90% of the job you would have to do manually. But, after using it 3-4 times I think it will be very improbable that we can release newer versions without having to do small manual tweaks before.

In any case, I'll add a prompt to show the portal revision and confirm, in the script. Just in case :+1:

Reply:
Sorry, I misunderstood the message. You mean using portal's downloaded `yarn` :+1: